pom.xml

```diff
@@ -6,15 +6,16 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.alliancegenome</groupId>
 	<artifactId>agr_mati</artifactId>
-	<version>0.0.0</version>
+	<version>0.0.1</version>
```

Comment:
@oblodgett You probably want to keep the default version nr reported to 0.0.0 te avoid any confusion when not updated to a current version.

Reply:
oops yes